packages/generator/templates/mutations/create__ModelName__.ts

```diff
@@ -1,20 +1,9 @@
 import { resolver } from "@blitzjs/rpc"
 import db from "__prismaFolder__"
-import {z} from "zod"
-
-if (process.env.parentModel) {
-  const Create__ModelName__ = z.object({
-    __parentModelId__: z.__parentModelIdZodType__(),
-    // template: __fieldName__: z.__zodType__(),
-  })
-} else {
-  const Create__ModelName__ = z.object({
-     // template: __fieldName__: z.__zodType__(),
-  })
-}
+import { __ModelName__Schema } from "../validations/schema"
```

Comment:
Our directory structure is, to place the schema in `./__modelName__/schema.ts`. This allows to import it in the `./__modelName__/mutations/*` files as well as in the Form.

And the zod schema is at the same place place all the other parts of the given model.

Reply:
Hey, @tordans, I think I agree this is design and have made the changes to the structure (Just FYI the previous version also had the schema in the __modelName__ forlder just one directory deeper in validations folder which I feel is redundant

The new folder structure would be like this: 

![image](https://user-images.githubusercontent.com/83594610/213470752-f3610605-61d8-4f7c-b694-387364765955.png)

which follow the accepted way in https://github.com/blitz-js/blitz/issues/2815#issuecomment-937942886